core/control-plane/control-plane-aggregate-services/src/main/java/org/eclipse/edc/connector/controlplane/services/ControlPlaneServicesExtension.java

```diff
@@ -99,6 +100,9 @@ public class ControlPlaneServicesExtension implements ServiceExtension {
 
     public static final String NAME = "Control Plane Services";
 
+    @Setting(key = "edc.policy.validation.enabled", description = "If true enable the policy validation when creating and updating policy definitions", defaultValue = "false")
```

Comment:
```suggestion
    @Setting(key = "edc.policy.validation.enabled", description = "If true enables the policy validation when creating and updating policy definitions", defaultValue = "false")
```

I'm also wondering if we want to set this disabled by default. I'd see more value in having it enabled by defaut

Reply:
For now i think we have to explicitly enable it, first because it's still part of the experimental period i think
and second most important if it's enabled by default it's a breaking change :)

Reply:
ok, but having an invalid policy in the system couldn't be considered as a malfunction?

Reply:
ok, so, let's merge! :man_dancing: